Mathlib/FieldTheory/Galois/Infinite.lean

```diff
@@ -267,4 +259,10 @@ theorem normal_iff_isGalois (L : IntermediateField k K) [IsGalois k K] :
   · simpa only [IntermediateFieldEquivClosedSubgroup, RelIso.coe_fn_mk, Equiv.coe_fn_mk,
       ← L.restrictNormalHom_ker] using MonoidHom.normal_ker (restrictNormalHom L)
 
+theorem isOpen_and_normal_iff_finite_and_isGalois (L : IntermediateField k K) [IsGalois k K] :
+    IsOpen L.fixingSubgroup.carrier ∧ L.fixingSubgroup.Normal ↔
+    FiniteDimensional k L ∧ IsGalois k L :=
+  ⟨fun h ↦ ⟨(isOpen_iff_finite L).mp h.1, (normal_iff_isGalois L).mp h.2⟩,
+   fun h ↦ ⟨(isOpen_iff_finite L).mpr h.1, (normal_iff_isGalois L).mpr h.2⟩⟩
```

Comment:
I don't think we need this since `by rw [isOpen_iff_finite, normal_iff_isGalois]` should work as a proof.

Reply:
It worked, thanks.